### PR TITLE
CNN with validation and error rate of 1.38%

### PR DESCRIPTION
**Pull request by alexa3204**
Visualization of misclassified data. Tested with 120 filters in convolutional block 2, 64 filters in convolutional block 1. Both resulted in a much larger error. 32 filters in convolutional block 1 worked similar to 64 filters, with 64 giving slightly better results. I could still test out different learning rates, loss functions, or optimizers (using RMS currently). Training set was reduced to 25,000 data with 20% for validation.

### CODE REVIEW

**Comment by alexa3204** [comment]
I might try out different parameters of the CNN, but this works pretty well.
